app/Console/Commands/UpdateSignupAndOrPostField.php

```diff
@@ -15,7 +15,7 @@ class UpdateSignupAndOrPostField extends Command
      *
      * @var string
      */
-    protected $signature = 'rogue:updatefield {target} {targetOldValue} {targetNewValue} {--signups} {--posts} {--logfreq=10000}';
+    protected $signature = 'rogue:updatefield {target} {targetOldValue} {targetNewValue} {--signups} {--posts} {--logfreq=10000} {--log}';
```

Comment:
we should set a default for the `log` option as well. probably just have it on by default.

Reply:
hm @sbsmith86 I'm not sure how to set the default to `true` by default - didn't seem to work when doing `{--log=true}`, `{--log=TRUE}`, or `{--log=1}`.

Reply:
Ack maybe we don't need it since it is just an on/off flag! Sorry. this looks great.

Reply:
No worries - thank you for the quick review! :)